util/cron/common-arkouda.bash

```diff
@@ -38,7 +38,7 @@ function test_release() {
   git checkout 1.24.1
   git checkout $currentSha -- $CHPL_HOME/test/
   git checkout $currentSha -- $CHPL_HOME/util/cron/
-  git checkout $currentSha -- $CHPL_HOME/util/test/perf/
+  git checkout $currentSha -- $CHPL_HOME/util/test/
```

Comment:
Why did we need this change?

Reply:
The old behavior impacted Arkouda benchmarks, so here this is checking out 1.24.1 for the release testing, but using the main branch's util/test directory (because we want this change to be used with Arkouda 1.24.1 testing)

Reply:
Ah, makes sense.